<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(88, 88, 88);
border-radius: 3px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="name">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>動漫名稱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="path">
       <property name="font">
        <font>
         <family>Microsoft JhengHei UI</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Path: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="folder_input">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
	font: 11pt &quot;Microsoft JhengHei UI&quot;;
 border: 1px solid ;
	color: rgb(222, 222, 222);

 border-radius: 10px;
border-color: rgb(255, 255, 255);
 selection-color: #F2F2F2;
}
QLineEdit:hover{
	font: 11pt &quot;Microsoft JhengHei UI&quot;;

 border: 1px solid rgb(19, 23, 255);.
border-color: rgb(255, 255, 255);

 border-radius: 10px;
color: rgb(222, 222, 222);
 selection-color: #F2F2F2
}</string>
       </property>
       <property name="placeholderText">
        <string>請輸入新資料夾名稱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="cancel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(250, 250, 250);
	background-color: rgb(102, 102, 102);
}
QPushButton:hover{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(102, 102, 102);
	background-color: rgb(250, 250, 250);
}</string>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirm">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(250, 250, 250);
	background-color: rgb(102, 102, 102);
}
QPushButton:hover{
	border-style :outset;
	border-color: rgb(26, 120, 179);
	border-width: 2px;
	border-radius: 10px;
	color: rgb(102, 102, 102);
	background-color: rgb(250, 250, 250);
}</string>
          </property>
          <property name="text">
           <string>確認資料夾名稱</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
